config/chime_science_run_gpu_no_output.yaml

```diff
@@ -737,7 +737,7 @@ gpu:
     #Copy out queue
     - name: hsaRfiBadInputOutput
     - name: hsaRfiOutput
-    #- name: hsaRfiVarOutput
+    - name: hsaRfiVarOutput
```

Comment:
This command object isn't really used by the current config, since the name/size of the variance array hasn't been updated in it.  The size won't actually change, but the output will come from the `rfi_chime_input_sum` since there needs to be a reduction across inputs (with masking).   For this stage of the testing I'd suggest commenting it out.